Library/Formula/git-when-merged.rb

```diff
@@ -0,0 +1,13 @@
+class GitWhenMerged < Formula
+  desc "Find the where a commit was merged in git."
```

Comment:
The sentence above is garbled. Maybe

    Find when a commit was merged in git

?